data/l-legren.json

```
@@ -8,9 +8,20 @@
       "icon": "FaInstagram"
     },
     {
-      "name": "Follow me on GitHub!",
+      "name": "This is my GitHub!",
       "url": "https://www.github.com/l-legren",
       "icon": "FaGithubAlt"
-    }
-  ]
+    },
+    {
+      "name": "Personal Webpage",
+      "url": "https://www-leret-dev-com.onrender.com/",
+      "icon": "MdWebAsset"
+    },
+    {
+      "name": "Connect me on LinkedIn",
+      "url": "https://www.linkedin.com/in/carlos-leret/",
+      "icon": "FaLinkedinIn"
+    },
```

Please remove this trailing comma here. 

```suggestion
    }
```